api/src/models.d.ts

```diff
@@ -72,3 +72,158 @@ export interface ParticipantActivityForProgram {
   program_id: number;
   participant_activities: ParticipantActivityCompletionStatus[];
 }
+
+export interface Question {
+  assessment_question_id: number;
+  id?: number;
```

Comment:
As mentioned previously, for all data types that have an `id` field, the `id` field should display first.